src/content/docs/2/guide/frontend/trunk.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Trunk
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Trunk is a tool for compiling and running Rust based frontends. Make sure you go through the checklist below to set Trunk up for your project.
+
+## Checklist
+
+- Use SSG, Tauri doesn't officially support server based solutions.
+- Use `0.0.0.0` so that your device can detect your frontend.
+- Use the fork of Trunk made by one of the Tauri core member which contains a couple of fixes to it. Hopefully we can get these merged upstream eventually.
```

Comment:
Can be? Probably. Will be? No idea
@amrbashir thoughts?

Reply:
1. I have already opened the PRs before deciding on a fork because it is taking a while to get them merged.
2. https://github.com/thedodd/trunk/pull/494 and https://github.com/thedodd/trunk/pull/500
3. Yeah only for mobile, if using only desktop whether it is 1.0 or 2.0, you can just use the original `trunk`

Reply:
Adjusted the phrasing on this. Let me know if you all are comfortable with how it's shown now.